src/test/java/StringTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2를 ,로 split 할 경우 1과 2로 잘 분리하는지에 대한 테스트")
+    void splitTest() {
+        String[] str = "1,2".split(",");
+        assertThat(str).containsExactly("1","2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).containsExactly("1","2");
```

한줄로 사용해도 가독성에 크게 문제는 없을것 같네요.